src/riak_cs_config.erl

```diff
@@ -402,6 +403,15 @@ read_before_last_manifest_write() ->
 region() ->
     get_env(riak_cs, region, ?DEFAULT_REGION).
 
+%% @doc This switch changes mapreduce input from listkeys to
+%% fold_objects after riak_kv's mapreduce input optimization
+%% merged and enabled. With fold_objects it will have 1/2 IO
+%% when scanning the whole bucket.
+-spec use_2i_for_storage_calc() -> boolean().
+use_2i_for_storage_calc() ->
+    riak_cs_list_objects_utils:fold_objects_for_list_keys()
```

Comment:
Actually, this was to ensure Riak is later than 1.4 - having 2i `<<"key">>` index. Maybe it can be removed.